config/auth.php

```diff
@@ -66,8 +66,8 @@
 
     'providers' => [
         'users' => [
-            'driver' => 'eloquent',
-            'model' => Rogue\Models\User::class,
+            'driver' => 'northstar',
+            'model' => App\User::class,
```

Comment:
I believe this should still be `Rogue\Models\User::class`. `App\User::class` is just provided as a placeholder in the readme for whatever the given application's user model is named.